services/cognito/spec/requests/pools_spec.rb

```diff
@@ -147,5 +154,11 @@
         end
       end
     end
+
+    private
```

Comment:
I think you should also take a look at rspec documentation:
https://relishapp.com/rspec/rspec-core/docs/example-groups/shared-examples

Reply:
If you really want to test that the attributes are there for all the requests you should use a shared example at minimum.
You could write a Resource test that ensures we are serializing the items as well, but then to be really accurate, you'd need to expect the specific resource class to be called. So i think you'd be better off with writing a shared example, or test it in one request since you're not even checking for the values of the attributes